pkg/trimaran/handler.go

```diff
@@ -0,0 +1,99 @@
+package trimaran
+
+import (
+	"fmt"
+	v1 "k8s.io/api/core/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientcache "k8s.io/client-go/tools/cache"
+	"sync"
```

Comment:
go fmt

Reply:
`go fmt` doesn't fix it, I fixed it manually.